packages/components/components/Blurb/Blurb.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { styled } from "../../stitches.config";
+import Button from '../Button';
+import Wrapper from "../Wrapper";
+import { ArrowRightIcon } from "@radix-ui/react-icons";
+
+export interface IBlurbProps {
+  heading: string;
+  body: string;
+  link?: { label: string, href: string };
+}
+
+export default function Blurb({ heading, body, link }: IBlurbProps) {
+  return (
+    <Container id="about">
+      <Wrapper>
+        <Content>
+          <h2>
+            {heading}
+          </h2>
+          <div>
+            <p>{body}</p>
+            {link && 
+              <Button as="a" type="link" href="https://creators.mirror.xyz/-lNPJRz2GLWIcsuMTZqklGNEWRrY7Nk0Y33Qn6Lw4q4"
```

Comment:
Ah, yeah. I added that additional check along with a fallback label should it not be provided.

Reply:
sorry to be clear, should this not be:

`<Button as="a" type="link" href={link.href}`

Otherwise, why define `link.href` as a parameter if it's not used?